cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -29,7 +29,7 @@ type Exception struct {
 
 var (
 	// Lists to memorize results required for the subsequent steps
-	ecsTaskList   = []ecs.Task{}
+	EcsTaskList   = []ecs.Task{}
```

Comment:
Ah right. But I'd still create the ecsTaskList variable in e2einstancesteps/common_steps.go. But I dont feel strongly because of the reason you mentioned.